packages/@okta/vuepress-site/docs/guides/submit-app/main/wfconnector/submit.md

```diff
@@ -0,0 +1,23 @@
+Submit a specific Workflows connector version from your Workflows Connector Builder environment in your org.
+
+#### Connector details
+
+* **Connector**: Select the Workflows connector you want to submit.
+
+* **Version**: Select the version of Workflows connector you want to submit.
+
+The list of available connectors and the associated versions are based on the Workflows Connector Builder environment from the org associated with your user credential. These fields are automatically populated if you submit from the Workflows Connector Builder.
```

Comment:
`These fields are automatically populated if you submit from the Workflows Connector Builder.` may not be true as it may be descoped for GA https://okta.slack.com/archives/C0391EXF0RZ/p1676492815620389?thread_ts=1676492239.757629&cid=C0391EXF0RZ